src/overview/cc-tile-instances.js

```diff
@@ -148,11 +150,11 @@ export class CcTileInstances extends LitElement {
         }
 
         .instances[data-type=running] {
-          --status-color: #2faa60;
+          --status-color: var(--color-legacy-green);
```

Comment:
This is temporary because text comes with an icon which color is still legacy.
See #407